modules/generate-verify/util/verify.sh

```diff
@@ -44,8 +44,7 @@ cleanup() {
 }
 trap "cleanup" EXIT SIGINT
 
-cp -a "${projectdir}/." "${tmp}"
-rm -rf "${tmp}/_bin" # clear all cached files
+rsync -aEq "${projectdir}/." "${tmp}" --exclude "_bin/"
```

Comment:
The `cp` command was throwing errors because the `_bin` folder was being changed while the `cp` was running.
Instead, we can use rsync, which has an exclude option for the `_bin/` folder.